WcaOnRails/app/assets/stylesheets/email.css.scss

```diff
@@ -4,6 +4,11 @@ h2.alert {
   color: red;
 }
 
+p.alert {
+  color: red;
+  font-weight: bold;
+}
+
 span.alert {
```

Comment:
Here instead of creating two rules with the same styles, you can just have the same behavior using:
```css
p.alert,
span.alert {
// the styles
}
```

and avoiding the code duplication :)

Reply:
That makes a lot of sense!